vdr/vdr.go

```diff
@@ -163,3 +171,50 @@ func (r *VDR) Deactivate(DID did.DID, current hash.SHA256Hash) {
 	logging.Log().Debugf("Deactivating DID Document: %s", DID)
 	panic("implement me")
 }
+
+func (r *VDR) resolveControllers(doc did.Document) ([]did.Document, error) {
+	return r._resolveControllers([]did.Document{doc})
+}
+
+func (r *VDR) _resolveControllers(input []did.Document) ([]did.Document, error) {
```

Comment:
why the construction with a hidden private method that accepts multiple docs and one private that accepts a single one?